### ISSUE TITLE
Move CSVY writer from custom abundance widget to io

### ISSUE THREAD

**Issue by Rodot-** [tardis-sn/tardis#1927]
**Is your feature request related to a problem? Please describe.**
Would be nice to have simpler access to the csvy writer for general model creation.  Currently it is a part of the custom abundance widget, but the tool is generally useful.

**Describe the solution you'd like**
Move this over to the io module to make access easier.

**Comment by umak1106** [comment]
Hello , where exactly in the io module do you want this to be moved over to ?
@Rodot-

**Comment by VedantGandhi04** [comment]
Hello @Rodot- Is this issue still open? If yes, I'm willing to contribte.

**Comment by Rodot-** [comment]
Yes, I believe it is technically still open though there is currently an old PR for this still open so it might be good to try another issue until we decide if we want to close the current open PR on this.  I'll ping the reviewers for it to find out and let you know!

**Comment by AyushiDaksh** [comment]
Hi @Rodot- can I take this up? I see the earlier PR related to this issue was closed without merging.

**Comment by Zrahay** [comment]
Heyo @Rodot- ! Understand that no work is being done on this issue, so can I take this up?
